test/end2end.py

```diff
@@ -320,11 +328,10 @@ def test_env_variables(self):
             self.assertTrue(key in out, "%s is not in out" % key)
 
         regex = r'.*-envlist [^ ]*USER.*'
-        self.assertTrue(regex.find(out), "Variablesprefix USER isn't passed to mympirun script env")
```

Comment:
ha, that's a nice one. how did this one ever pass?

Reply:
```
>>> 'foo'.find('bar')
-1
>>> bool('foo'.find('bar'))
True
```

Reply:
`-1` should have guessed that.